packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -68,4 +70,60 @@ export class AccountDeleteManager {
     }
     this.log = Container.get(AuthLogger);
   }
+
+  public async deleteAccountDb(accountRecord: Account) {
```

Comment:
Create a destroy function that does the same work as the `/account/delete` route. Minus all the session checking, rate limiting, emiting metrics.

This method will be used in both the cloud task and delete route.